pkg/controller/directvolumemigration/rsync.go

```diff
@@ -645,6 +672,19 @@ func (t *Task) getRsyncRoute(namespace string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	// Check if we can find the admitted condition for the route
+	admitted := false
+	for _, ingress := range route.Status.Ingress {
+		for _, condition := range ingress.Conditions {
+			if condition.Type == routev1.RouteAdmitted {
+				admitted = true
+				break
+			}
+		}
+	}
+	if !admitted {
```

Comment:
How soon does this run after route creation? Is there room for a race condition here where the route controller could be slow to react and we fail a migration due to that?

Reply:
+1 yes, this is not something we should fail at. Probably follow the same pattern if the route is not admitted within 10 mins, raise a warning like we do for pending pods that will be bubbled up to the UI

Reply:
+1 good point. Will add a new task to the itinerary.